modules/serverless-rn-1-14-0.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-14-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} 1.14.0
+
+[id="new-features-1-14-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.20.0.
+* {ServerlessProductName} uses Knative Eventing 0.20.0.
+* {ServerlessProductName} now uses Kourier 0.20.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.20.0.
+* {ServerlessProductName} now uses Knative Kafka 0.20.0.
+* Apache Kafka on {ServerlessProductName} is now Generally Available (GA).
++
+[IMPORTANT]
+====
+Only the `v1beta1` version of the API for Apache Kafka on {ServerlessProductName} is supported. Do not use the `v1alpha1` version of the API, as this is deprecated.
+====
+* The Operator channel for installing and upgrading {ServerlessProductName} has been updated to `stable` for {product-title} 4.6 and newer versions.
+* {ServerlessProductName} is now supported on IBM Power Systems, IBM Z, and LinuxONE, except for the following features, which are not yet supported:
```

Comment:
LinuxONE? Where is this coming from? I'm hearing this for t he first time :)

Reply:
It came from @SNiemann15 IIRC, should we remove it?